core/templates/pages/login-page/login-page.component.ts

```diff
@@ -62,6 +62,8 @@ export class LoginPageComponent implements OnInit {
       let authSucceeded = false;
       try {
         authSucceeded = await this.authService.handleRedirectResultAsync();
+      // Unknown type is used because we don't know which type of error is
+      // thrown.
```

Comment:
There should be `isinstance` to verify the type of error below

Reply:
Try, but it causes failure inside lighthouse.

![Screenshot from 2023-02-27 17-44-58](https://user-images.githubusercontent.com/80846488/221561832-82048290-1120-4002-ad3e-a9a7dc02fda5.png)

Reply:
Sorry nto sure what you mean.